config/printer-rf2000v2-single.cfg

```diff
@@ -0,0 +1,235 @@
+# Configuration for the Renkforce RF2000V2 SINGLE 3D printer
+# Note: This configuration is not for RF2000 and not for the RF1000
+# Note: This is work in progress, not yet ready for production!
+
+# Attention: This file is configured for use with an E3D v6 hotend instead of the original Renkforce v3 hotend!
```

Comment:
Can you please change this so it works for an unmodified RF2000v2? It is supposed to be a generic config file, not an example which just works for your copy...